docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
> It's working, however you have to be careful about this rules. Moving the server, I had to change the host name and such settings are very likely to be missed.

Migrations are always tough. :)

> That is bad. You wrote in a different post, that you run sa-exim before, you are sure that you run spamassassin as the same user with the same settings? Or you had more ACL-checks in use which are not part of the current vexim setup (SPF, rDNS checks, ...)?

Here's what I found in sa-exim config file:
```
# SAspamcUser: The username passed to spamc. Some tricks are needed to 
# decide on one user when there are many recipients. This string is of
# course expanded. If unset or empty, spamc will use the user Exim 
# runs as. We suggest that you decide what username to use in the ACLs
# and set an ACL variable.
#SAspamcUser: $acl_m2
```

So it seems my spam checks were performed as Debian-exim due to this variable being unset, and the statuses of the two .spamassassin folders seem to confirm this guess ("Rgp" means August):
```
# ls ~Debian-exim/.spamassassin/ -l
viso 87708
-rw------- 1 Debian-exim Debian-exim 21008384 Lap  1  2014 auto-whitelist
-rw------- 1 Debian-exim Debian-exim        6 Lap  1  2014 auto-whitelist.mutex
-rw------- 1 Debian-exim Debian-exim     6288 Rgp 17 17:07 bayes_journal
-rw------- 1 Debian-exim Debian-exim     1956 Rgp 17 17:06 bayes.mutex
-rw------- 1 Debian-exim Debian-exim 83865600 Rgp 17 16:52 bayes_seen
-rw------- 1 Debian-exim Debian-exim  5332992 Rgp 17 17:06 bayes_toks
-rw-r--r-- 1 Debian-exim Debian-exim     1487 Lap 24  2009 user_prefs
# ls -l ~debian-spamd/.spamassassin/
viso 2140
-rw------- 1 debian-spamd debian-spamd       6 Rgp 22 18:45 bayes.mutex
-rw------- 1 debian-spamd debian-spamd   86016 Rgp 22 18:45 bayes_seen
-rw------- 1 debian-spamd debian-spamd 2633728 Rgp 22 18:45 bayes_toks
drwxr-xr-x 2 debian-spamd debian-spamd    4096 Bir  6  2015 sa-compile.cache
-rw-r--r-- 1 debian-spamd debian-spamd    1869 Rgp 18 18:19 user_prefs
```

I guess I'll just try copying over all files from `~Debian-exim/.spamassassin` to `~debian-spamd/.spamassassin`? :)